manage-teams-identity-mobile-and-desktop/README.md

```diff
@@ -0,0 +1,46 @@
+---
+page_type: sample
+languages:
+- python
+products:
+- azure
+- azure-communication-services
+---
+
+# Create and manage Communication access tokens for Teams users in mobile and desktop applications
+
+This code sample walks you through the process of acquiring a Communication Token Credential by exchanging an Azure AD token of a user with a Teams license for a valid Communication access token.
+
+This sample application utilizes the [MSAL Python](https://github.com/AzureAD/microsoft-authentication-library-for-python) library for authentication against the Azure AD and acquisition of a token with delegated permissions. The token exchange itself is facilitated by the `azure-communication-identity` package.
+
+To be able to use the token for Calling, use it to initialize the `CommunicationTokenCredential` from the `azure-communication-chat` library.
+
+## Prerequisites
+
+- An Azure account with an active subscription. [Create an account for free](https://azure.microsoft.com/free/?WT.mc_id=A261C142F).
+- Python 3.7 or later version.
+- An active Communication Services resource and connection string. [Create a Communication Services resource](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource/).
+- Azure Active Directory tenant with users that have a Teams license.
+
+## Before running sample code
+
+1. Complete the [Administrator actions](https://docs.microsoft.com/azure/communication-services/quickstarts/manage-teams-identity?pivots=programming-language-javascript#administrator-actions) from the [Manage access tokens for Teams users quickstart](https://docs.microsoft.com/azure/communication-services/quickstarts/manage-teams-identity).
+   - Take a not of Fabrikam's Azure AD Tenant ID and Contoso's Azure AD App Client ID. You'll need the values in the following steps.
```

Comment:
```suggestion
   - Take a note of Fabrikam's Azure AD Tenant ID and Contoso's Azure AD App Client ID. You'll need the values in the following steps.
```

Reply:
@AikoBB can you pls create a PR for .NET & JS as well?

Reply:
for JS - https://github.com/Azure-Samples/communication-services-javascript-quickstarts/pull/82

Reply:
for .net - https://github.com/Azure-Samples/communication-services-dotnet-quickstarts/pull/50